stl/BPCore@0.1.0.sty

```diff
@@ -52,31 +52,21 @@ data AnchorMerkleTreeOpretProof : mpcProof CommitVerify.MerkleTree, dbcProof Opr
 @mnemonic(europe-mister-imitate)
 data AnchorMerkleTreeTapretProof : mpcProof CommitVerify.MerkleTree, dbcProof TapretProof
 
-@mnemonic(report-process-stuart)
-data BlindSealTxPtr    : method Method
-                       , txid TxPtr
+@mnemonic(metro-mars-canada)
+data BlindSealTxPtr    : txid TxPtr
                        , vout Bitcoin.Vout
                        , blinding U64
 
-@mnemonic(baby-region-proxy)
-data BlindSealTxid     : method Method
-                       , txid Bitcoin.Txid
+@mnemonic(halt-crash-valid)
+data BlindSealTxid     : txid Bitcoin.Txid
                        , vout Bitcoin.Vout
                        , blinding U64
 
-@mnemonic(meaning-extra-sherman)
-data ExplicitSealTxPtr : method Method
-                       , txid TxPtr
-                       , vout Bitcoin.Vout
-
-@mnemonic(june-total-denver)
-data ExplicitSealTxid  : method Method
-                       , txid Bitcoin.Txid
-                       , vout Bitcoin.Vout
-
-@mnemonic(bali-boris-plasma)
-data Method            : opretFirst | tapretFirst
+@mnemonic(velvet-neptune-bazooka)
+data ExplicitSealTxPtr : txid TxPtr, vout Bitcoin.Vout
 
+@mnemonic(chicago-europe-phantom)
+data ExplicitSealTxid  : txid Bitcoin.Txid, vout Bitcoin.Vout
```

Comment:
This type becomes strictly equal to `Outpoint`. I do not think we need it anymore

Reply:
I think we do need it because `ExplicitSeal` can be `ExplicitSeal<Txid>` or `ExplicitSeal<TxPtr>`. Or do you think we could drop the generic from `ExplicitSeal`?

Reply:
I think we can drop generic form and just implement the trait we need (something like `TxoSeal`... there were some traits for explicit seals) and have `ExplicitSeal<TxPtr>` renamed into `GraphSeal` (which is a type alias to it anyway, so we do not need both) - and `ExplicitSeal<Txid>` (with its alias `GenesisSeal`) removed, and `Outpoint` uses instead of them.

We can make things even more simple: in v0.12 I got rid of `TxPtr` taking convention that `Txid` of zeros means witness output-based seal. This reduces a lot of code here, at no risk cost (from my experience, each time you have an enum in core, add 1000 lines at least where you deal with it's `match`es....)

Reply:
I think here I can say the same I said in https://github.com/RGB-WG/rgb-core/pull/285#issuecomment-2597812903: since this is just refactor I'll do this in another set of PRs after these ones are ACKed

Reply:
I am taking my suggestion back: single type of seals doesn't work and I had to differentiate two seal types it in v0.12: https://github.com/BP-WG/bp-core/pull/112